sklearn/manifold/tests/test_isomap.py

```diff
@@ -15,65 +10,75 @@
 from sklearn import preprocessing
 from sklearn.datasets import make_blobs
 from sklearn.metrics.pairwise import pairwise_distances
-from sklearn.utils._testing import assert_allclose, assert_allclose_dense_sparse
-
+from sklearn.utils._testing import (
+    assert_almost_equal,
+    assert_allclose,
+    assert_allclose_dense_sparse,
+    assert_array_equal,
+)
 from scipy.sparse import rand as sparse_rand
 
 eigen_solvers = ["auto", "dense", "arpack"]
 path_methods = ["auto", "FW", "D"]
 
+# TODO: Isomap fit/fit_transform must preserve dtype
+
```

Comment:
you can now remove this comment

Reply:
Done as part of https://github.com/scikit-learn/scikit-learn/commit/1d72c03b20d3e0e37d48c0837d8f58c3d27654bf.